index.js

```diff
@@ -134,51 +114,32 @@ function nowServing(line) {
   }
   return `Currently serving ${line.shift()}.`
 }
-
-const line = []
-
-// ***** Scope & Closures - Question 1 *****
-
-
-// *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
-
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 1`
-
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 2`
-
+// const line = []
+// console.log(takeANumber(line, "Loren"))
+// console.log(takeANumber(line, "Liza"))
 // console.log(nowServing(line))
-// // => `Currently serving 1.`
-
 // console.log(nowServing(line))
-// // => `Currently serving 2.`
-
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 3`
 
-// console.log("%c----------", "color: red") 
-
-// ***** Scope & Closures - Question 2 *****
-// *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 2", "color: red")
-
-// const newLine = []
-// const takeATicketNumberFunc = ticketNumberGeneratorFunc()
-// console.log(takeATicketNumberFunc(newLine))
-// // => `Welcome. You are ticket number 1`
-
-// console.log(takeATicketNumberFunc(newLine))
-// // => `Welcome. You are ticket number 2`
+//Scope & Clos Q1
+function takeATicketNumber(lineArr) {
+  //let tickNum = 1
+  let tickNum = lineArr.length
+  if (!lineArr.length) {
+    return "Welcome. You are ticket number 1"
+  } 
+  return `"Welcome. You are ticket number ${tickNum}`
+}
```

Comment:
If we move line 125 to line 123. Then the variable tickNumber would be in the global scope and we could just increment that global variable for every ticket that is taken.